### PR TITLE
Add creation_date metrics

### PR DESCRIPTION
**Pull request by capuche2412**
Hello,

This PR has the purpose to add a new metrics :
`elasticsearch_indices_settings_creation_timestamp_seconds`

The motivation was that we want to keep tracks of the index creation dates. We compare it with log timestamp and create an alert if logs are timestamped in the future or in the past

### CODE REVIEW

File: collector/indices_settings.go [hunk 1]
```diff
@@ -105,6 +106,21 @@ func NewIndicesSettings(logger log.Logger, client *http.Client, url *url.URL) *I
 					return val
 				},
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "indices_settings", "creation_date"),
```
Comment (SuperQ) [inline collector/indices_settings.go:112]: This needs to follow Prometheus naming practices.

```suggestion
					prometheus.BuildFQName(namespace, "indices_settings", "creation_timestamp_seconds"),
```

File: README.md [hunk 2]
```diff
@@ -187,6 +187,7 @@ Further Information
 | elasticsearch_indices_search_query_total                             | counter    | 1           | Total number of queries                                                                             |
 | elasticsearch_indices_segments_count                                 | gauge      | 1           | Count of index segments on this node                                                                |
 | elasticsearch_indices_segments_memory_bytes                          | gauge      | 1           | Current memory size of segments in bytes                                                            |
+| elasticsearch_indices_settings_creation_date                         | gauge      | 1           | Index setting value for index.creation_date                                                         |
```
Comment (SuperQ) [inline README.md:190]: This should be more descriptive, and should match the Help text.

Maybe something like "Timestamp of the index creation".

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Review by SuperQ — APPROVED** [review]

File: collector/indices_settings.go [hunk 3]
```diff
@@ -105,6 +106,21 @@ func NewIndicesSettings(logger log.Logger, client *http.Client, url *url.URL) *I
...
+					prometheus.BuildFQName(namespace, "indices_settings", "creation_timestamp_seconds"),
+					"index setting creation_date",
+					defaultIndicesTotalFieldsLabels, nil,
+				),
+				Value: func(indexSettings Settings) float64 {
+					val, err := strconv.ParseFloat(indexSettings.IndexInfo.CreationDate, 64)
+					if err != nil {
+						return float64(defaultDateCreation)
+					}
+					return val
```
Comment (SuperQ) [inline collector/indices_settings.go:121]: I was just looking into this field, it looks like the value is actually expressed as a timestamp in epoch millseconds. So we need to convert this.

```suggestion
					return val/1000.0
```

**Review by SuperQ — CHANGES_REQUESTED** [review]

File: collector/indices_settings.go (hunk 3 above)
Comment (capuche2412) [reply collector/indices_settings.go]: I also precise the unit in the README description

**Review by SuperQ — APPROVED** [review]

**Review by sysadmind — APPROVED** [review]

**Pull request merged**